.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy
+on:
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+  deploy:
+    name: build dependencies & create artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.0.0
+      - name: Build JS dependencies
+        uses: PrestaShopCorp/github-action-build-js/12@v1.0
+        with:
+          cmd: yarn
+          path: ./_dev
+      - name: Install composer dependencies
+        run: composer install --no-dev -o
+      - name: Clean-up project
+        uses: PrestaShopCorp/github-action-clean-before-deploy@v1.0
+      - name: Create & upload artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: ps_checkout
+          path: ../
+      - name: Publish to GitHub Release
+        if: github.event_name == 'release' && github.event.action == 'published'
+        uses: actions/upload-release-asset@v1.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.release_info.outputs.upload_url }}
```

Comment:
Not sure this is going to work if release-drafter is not run this the same workflow

Reply:
I don't know, I just saw on the github action repo that it worked like this.

Reply:
it will because of the event check above, the release is already published